HW3/taniakhurd/H_w_3.2.py

```diff
@@ -0,0 +1,14 @@
+n = 1234
+n1 = 1 * 2 * 3 * 4
+print(n1)
+
+b = input(int())
+b1 = ''
+for i in b:
+    b1 = i + b1
+print(b1)
+
+a = 3254
+b = sorted(a)
+sorted_string = ''.join(b)
+print(sorted_string)
```

Comment:
number = input("Enter a four digit number:")

print(f'The product of numbers is: {int(number[0]) * int(number[1]) * int(number[2]) * int(number[3])}')

print(f'Reversed number: {number[::-1]}')

print(f'Sorted list: {list("".join(sorted(number)))}\n')